open-company/open-company-web
# FoC refresh: desktop and mobile

Card: https://trello.com/c/Q1uWzWoG

To test:
- from desktop
- add a post
- [x] share it
- click ellipsis menu
- [x] edit it
- [x] delete it
- [x] move it to another board
- now add some attachments to the post
- [x] do you see only one line of attachments when the post is collapsed? Good
- [x] do you see all of them when the post is expanded? Good
- now switch to mobile
- [x] attachments ok? Good
- [x] ellipses menu is always visible up top? Good
- [x] menu items work? Good
